README.md

```diff
@@ -19,15 +19,21 @@ Configuration
 
 It is required that a config.yaml exist in the root of the repository.  This will provide information for connecting to AWS.
 
+Note: If you are using Elasticsearch via AWS and your Elasticsearch domain is restricted use the last four lines of the example config to support IAM authentication. Otherwise emit the aforementioned config lines.
+
     buckets:
         bucket_name:
             accessKey: XXXXXXXXXXXXXXXXXXXX
             secretKey: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
         bucket_name_2:
             accessKey: XXXXXXXXXXXXXXXXXXXX
             secretKey: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
-    elasticSearchHost:
-        - localhost:9200
+    elasticsearch:
+        connectionString: http://localhost:9200/index
+        aws: True
```

Comment:
I may be misunderstanding.  What I meant was that "aws: True" shouldn't exist.  We can determine this based off of accessKey and secretKey (and possibly region?) right?